webapp/src/ts/effects/reports.effects.ts

```diff
@@ -334,9 +334,15 @@ export class ReportsEffects {
             .get(report.doc._id)
             .then(existingRecord => {
               this.reportActions.setFirstSelectedReportDocProperty({ _rev: existingRecord._rev });
```

Comment:
I don't think this line is necessary anymore. It was useful before when we actually pushed the doc we had in the state, but right now it just populates the state with a rev that's about to become irrelevant once we put in the next line. 
Also there's another line in the same file where we only minify the contact lineage. That can also be removed.

Reply:
I wasn't sure if it still made sense to keep this line so thanks for clarifying :+1: